dtale/static/css/main.css

```diff
@@ -8747,6 +8747,16 @@ caption {
   text-align: center;
 }
 
+.pinned-data-viewer-menu {
+  top: 100%;
```

Comment:
If the menu is taller than the screen, the bottom menu items are not styled properly -- need to fix that